rtf_vis_tool/src/Components/CoordinateGrid.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import {Line, Html} from "drei";
+
+//Function that creates the grid lines along with xy plane
+const getGridLines = () => {
+    var coordPairSet = []
+    //add x grid lines
+    for (var z = -10; z <= 10; z++) {
+      coordPairSet.push([[-10, 0, z], [10, 0, z]]);
+    }
+    //add z grid lines
+    for (var x = -10; x <= 10; x++) {
+      coordPairSet.push([[x, 0, -10], [x, 0, 10]]);
+    }
+
+    var finalLineGridSet = []
+    coordPairSet.map(coordPair => {
+      finalLineGridSet.push(<Line points={coordPair} color="black" position={[0,0,0]} lineWidth={0.1}/>);
+    });
+    return finalLineGridSet;
+}
+
+//Component which renders the x,y,z axes along with the grid lines along xy plane
+const CoordinateGrid = () => { 
```

Comment:
could do the docstring be more detailed -- are we returning html here? seems nonstandard for js?

Reply:
do you think this can go into a config? that will solve the magic numbers issue?

Reply:
I have added a more detailed docstring about the return type. Specifically, every single JS file inside the `Components` directory returns the format JSX, which is essentially a modified version of HTML that is meant to be readable by React code. So while it looks like HTML, it's technically not HTML. It's a syntax that React requires to render it's components. I can probably add this description into the README.md file, since JSX format applies to every file I have in the `Components` folder.